docs/cli.rst

```diff
@@ -43,6 +43,30 @@ and providing a simple YAML configuration file:
 The ``LOGGING`` entry is passed to :func:`logging.config.dictConfig` and should conform to the schema for that call.
 
 
+Environment variable substitution
+---------------------------------
+YAML configuration files have basic support for environment variables.
+You can use bash style syntax: ``${ENV_VAR}`` optionally you can provide default values ``${ENV_VAR:default_value}``
+
+
+.. code-block:: yaml
+
+    # foobar.yaml
+    AMQP_URI: amqp://${RABBITMQ_USER:guest}:${RABBITMQ_PASSWORD:password}@${RABBITMQ_HOST:localhost}
+
+Run your service with environment variables set:
+
+.. code-block:: shell
+
+    $ RABBITMQ_USER=user RABBITMQ_PASSWORD=password RABBITMQ_HOST=host nameko run --config ./foobar.yaml <module>[:<ServiceClass>]
+
+If you need to quote your yaml values explicit resolver ``!env_var`` is required
```

Comment:
If you need to quote the values in your YAML file, the explicit ``!env_var`` resolver is required: